src/cli/commands/beacon.ts

```diff
@@ -34,18 +35,23 @@ export class BeaconNodeCommand implements CliCommand {
       .option("-eth1, --eth1RpcUrl [url]", "Url to eth1 rpc node")
       .option("--rpc [api]", "Exposes the selected RPC api, must be comma separated")
       .option("-c, --configFile [config_file]", "Config file path")
+      .option("-l, --loggingLevel [DEBUG|INFO|WARN|ERROR]", "Logging level")
```

Comment:
maybe generate options from LogLevel enum instead of hardcoding?

Reply:
Subsequent PR :)

Reply:
See #218